src/components/pages/tutorials/Tutorials.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Icon from '../../layout/Icon/Icon'
+
+const Tutorials = () => (
+  <main className="tutorials">
+    <Icon
+      alt="calendrier"
```

Comment:
```suggestion
      alt=""
```
Pas besoin d'alternative pour un picto, ça n'apporte pas de plu value à l'utilisateur.